src/api/response/iiif/manifest.js

```diff
@@ -259,6 +256,37 @@ function transform(response) {
       }
     }
 
+    /** Add logo manually (w/o IIIF Builder) */
+    const nulLogo = {
+      id: "https://iiif.dc.library.northwestern.edu/iiif/2/00000000-0000-0000-0000-000000000003/full/pct:50/0/default.webp",
+      type: "Image",
+      format: "image/webp",
+      height: 139,
+      width: 1190,
+    };
+    jsonManifest.logo = [nulLogo];
+
+    /** Add provider manually (w/o IIIF Builder) */
+    const provider = {
+      id: "https://dc.library.northwestern.edu/",
+      type: "Agent",
+      label: { en: ["Northwestern University Libraries"] },
```

Comment:
```suggestion
      label: { none: ["Northwestern University Libraries"] },
```

We go with `none` elsewhere.